### PR TITLE
add fprintd support

### PR DESCRIPTION
**Pull request by abbradar**
This adds support for fingerprint scanners in NixOS (with a pam module). git version of libfprint has much broader support of scanners, so we can use it as a default.

### CODE REVIEW

File: pkgs/development/libraries/libfprint/master.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    rev = "35e356f625d254f44c14f720c0eb9216297d35c2";
+    sha256 = "b7fd74a914d7c4e2999ac20432a7f2af5d6c7af5e75a367bc3babe03e4576c86";
+  };
+
+  patches = [ ./0001-lib-Add-VFS5011-driver.patch ];
+
+  buildInputs = [ libusb glib nss nspr pixman ];
+  nativeBuildInputs = [ libtool autoconf automake113x pkgconfig ];
+
+  configurePhase = ''
```
Comment (lethalman) [inline pkgs/development/libraries/libfprint/master.nix]: Use `configureScript = "./autogen.sh";`

Comment (abbradar) [inline pkgs/development/libraries/libfprint/master.nix]: Thanks, fixed.

**Comment by lethalman** [comment]
Travis passed. Didn't test the nixos module though. Merged thanks.

**Comment by abbradar** [comment]
Works for me, at least (with usual fprintd problems). Thanks for merging!

**Pull request merged**